File: Hyunseok-Kim/baekjoon/python/boj_17256.py
# https://www.acmicpc.net/problem/17256

a_value = input()
c_value = input()
a_x, a_y, a_z = a_value.split(' ')
c_x, c_y, c_z = c_value.split(' ')

b_x = (int(a_z) - int(c_x)) * -1
b_y = int(int(c_y) / int(a_y))
b_z = (int(a_x) - int(c_z)) * -1

print(f"{b_x} {b_y} {b_z}")

File: Changhun-Han/src/baekjoon/beginner100/BOJ_17256.java
package baekjoon.beginner100;

import java.util.Scanner;

/**
 * @see <a href="https://www.acmicpc.net/problem/17256">
 * https://www.acmicpc.net/problem/17256
 * </a>
 */
public class BOJ_17256 {
    
    public static void main (String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int[] a = { scanner.nextInt(),
                    scanner.nextInt(),
                    scanner.nextInt() };
        
        int[] c = { scanner.nextInt(),
                    scanner.nextInt(),
                    scanner.nextInt() };
        
        int[] b = { c[0] - a[2],
                    c[1] / a[1],
                    c[2] - a[0] };
        
        System.out.printf("%s %s %s", b[0], b[1], b[2]);
        scanner.close();
    }
    
}